service_test.go

```diff
@@ -494,6 +576,11 @@ func (mln *MockLn) SendPaymentSync(ctx context.Context, senderPubkey string, pay
 	return "123preimage", nil
 }
 
+func (mln *MockLn) SendKeysend(ctx context.Context, senderPubkey string, amount int64, destination, preimage string, custom_records []TLVRecord) (preImage string, err error) {
+	//todo more advanced behaviour
```

Comment:
what advanced behaviour is needed?

Reply:
Not sure, it's mentioned in SendPaymentSync, so mentioned here. Perhaps multiple users?

Reply:
I think that will be a different method, so maybe we can just remove these TODO comments

Reply:
Done!